src/App.tsx

```diff
@@ -14,11 +15,121 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const App: React.FC = () => {
+  const [confirmBtn, setState] = useState(true);
+  const [selectedOption, setSelect] = useState(1);
```

Comment:
```suggestion
  const [confirmBtn, setconfirmBtn] = useState(true);
  const [selectedOption, setSelectedOption] = useState(1);
```
Назва функції для установки стейту повинна починатись із set а далі має бути назва того значення, яке міняється